packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -197,22 +194,17 @@ local function fetch_and_store_marathon_apps(auth_token)
        end
 
        local ports = task["ports"] --task host port mapping by default
-       if is_ip_per_task(app) then
-         ports = {}
-         if is_user_network(app) then
+
+       if is_container_network(app) then
             -- override with ports from the container's portMappings
-            local port_mappings = app["container"]["docker"]["portMappings"] or app["portDefinitions"] or {}
-            local port_attr = app["container"]["docker"]["portMappings"] and "containerPort" or "port"
+            ports = {}
+
+            local port_mappings = app["container"]["portMappings"] or app["portDefinitions"] or {}
```

Comment:
there won't be any `portDefinitions` if the app is running on a container network